api/registrations/views.py

```diff
@@ -96,10 +109,27 @@ def get_default_odm_query(self):
         query = base_query & permission_query
         return query
 
+    def is_blacklisted(self, query):
+        for query_param in query.nodes:
```

Comment:
I'm unclear on how this works; isn't `query' a modm queryset? Where does `query.nodes` come from?